internal/pkg/cli/create_test.go

```diff
@@ -21,13 +19,7 @@ import (
 
 func TestInteractiveCreateConfig(t *testing.T) {
 	// Create virtual console
-	var stdout io.Writer
-	if testhelper.TestIsVerbose() {
-		stdout = os.Stdout
-	} else {
-		stdout = io.Discard
-	}
-	c, state, err := testhelper.NewVirtualTerminal(expect.WithStdout(stdout), expect.WithDefaultTimeout(15*time.Second))
+	c, state, err := testhelper.NewVirtualTerminal(expect.WithStdout(testhelper.VerboseStdout()), expect.WithDefaultTimeout(15*time.Second))
```

Comment:
Ak sa spusti `make tests-verbose`, tak sa loguje viac vystupu do terminalu, a aj vystup interaktivnych testov, ktore simuluju pracu uzivatela.

Vyzeralo to nejak takto (`make tests-verbose`)
![image](https://user-images.githubusercontent.com/19371734/136735379-d3d44b6d-7306-4c75-b9d4-23775a5fad93.png)

CLI kniznica tam pouziva tzv. [ANSI Escape Codes](https://en.wikipedia.org/wiki/ANSI_escape_code#:~:text=ANSI%20escape%20sequences%20are%20a,character%2C%20are%20embedded%20into%20text.), napr. na definovanie farby, alebo zmazanie celeho riadku.

S tym doteraz nebol problem, presmerovalo sa to do terminalu testov a zobrazilo sa to.

No paralelne testy to rozbijaku, kedze sa do terminala zapisuje paralelne, ... co funguje spravne z beznym textom, ale blbne s ANSI kodmi.

Preto sa tieto kody pred zalogovanim odstranuju, vysledok je:
![image](https://user-images.githubusercontent.com/19371734/136735599-b4fcb2ed-9b36-42a0-bb5a-dd2d560d7c9b.png)

Tj, oproti predchadzajucemu sa loguje vzdy novy vystup (riadok sa neda zmazat) a bez farieb, ale to iba preto, ze sa ANSI kody odstranili.